_pages/en/it-picture-long-term.md

```diff
@@ -38,59 +47,49 @@ They can also use the digital services on behalf of clients.
+Employers, Banks, and Credit Unions are the new front lines of ESDC as they are now provided with technological means to interact with ESDC services.
 
-## Private Sector
+Ecosystem of services are created by leveraging private sector expertise where opportunities exist (win-win public-private partnerships), e.g.:
+- Banks detecting missed paycheck, job websites targeting people who are job hunting...
+- Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
 
-Any platform or device can be integrated with GC and ESDC services.
-This is possible because ESDC services are available as APIs that can be used by third party platforms and services (Banks, CRA, Social media, Job boards, Internet of things, ..).
+### Open Collaboration
 
-- Create ecosystem of services by leveraging private sector expertise where opportunities exist (win-win public-private partnerships)
-  - E.g., banks detecting missed paycheck, job websites targeting people who are job hunting
-  - Enhanced security through integration in private sector service (e.g., banks detecting fraudulent transactions)
+- ESDC custom built software are published as open source, allowing Academia and learning institutions to use them, helping grow technological skill sets in the country, fostering economic growth, improving transparency and trust in government, and increasing the breadth of talent for ESDC to use by welcoming contributions back
+- ESDC data (performance and automated decision making statistics) published as open datasets
 
 ## IITB-ESDC
 
 ### Modern Open Organization
 
 IITB is a modern open organization with a very open culture.
 Wherever possible code and projects are developed in the open and use of open source software is the norm.
+IITB is the digital, IM and IT advisor and enabler for ESDC and part of major business decisions.
+IITB is responsible for providing the automated services required to run a secure and agile business.
+This includes, automated software onboarding approval, key data exchange systems with which business may interact with, and an environment by which business may securely deploy their own solutions.
 
-### Strong culture of learning
-
-TODO: Split in two (IITB and Partners/Clients)
-
-- IITB is the digital, IM and IT advisor and enabler for ESDC
-- IITB is part of major business decisions
 - 90% of employees are able to work from anywhere (TODO: validate measure with trends)
-- IITB is a modern open organization with a very open culture that values learning (TODO: find a tangible example)
-- All internal and external services are available through APIs
+- Internal and external services are available through APIs
 - 98% of services are running on public cloud infrastructure (TODO: find cloud adoption rates)
-
-## Government Partners
```

Comment:
I think we need to keep this sub-section. This is where we need to refer to our other federal government partners (PSPC, SSC, Citizenship and Immigration, StatsCan, ISED, etc.)

Reply:
We decided to remove it since OneGC will make partnerships with individual departments less visible and will not be directly related to our vision for IITB-ESDC.  We added the relevant partnerships to each sections.  Banks and priivate sector.

I can add it back and we can see what we end up putting here